cpp/ql/test/library-tests/dataflow/external-models/validatemodels.expected

```diff
@@ -0,0 +1,4 @@
+| Dubious namespace "boost::asio" in sink model. |
+| Dubious namespace "boost::asio" in source model. |
+| Dubious namespace "boost::asio" in summary model. |
+| Invalid kind "remote-sink" in sink model. |
```

Comment:
I think we need to add `remote-sink` to [ModelValidation.qll](https://github.com/github/codeql/blob/main/shared/mad/codeql/mad/ModelValidation.qll)